src/main/java/cceclipseplugin/editor/DocumentManager.java

```diff
@@ -174,16 +177,16 @@ public void handleNotification(Notification n) {
 		// TODO(wongb): Build patch reorder buffer, making sure that they are applied in
 		// order.
 		// This is a temporary fix.
-		if (changeNotif.fileVersion <= fileMeta.getVersion()) {
-			try {
-				System.out.printf(
-						"ChangeNotification version was less than or equal to current version: %d <= %d; Notification: ",
-						changeNotif.fileVersion, fileMeta.getVersion(), new ObjectMapper().writeValueAsString(n));
-			} catch (JsonProcessingException e) {
-				e.printStackTrace();
-			}
-			return;
-		}
+//		if (changeNotif.fileVersion <= fileMeta.getVersion()) {
+//			try {
+//				System.out.printf(
+//						"ChangeNotification version was less than or equal to current version: %d <= %d; Notification: ",
+//						changeNotif.fileVersion, fileMeta.getVersion(), new ObjectMapper().writeValueAsString(n));
+//			} catch (JsonProcessingException e) {
+//				e.printStackTrace();
+//			}
+//			return;
+//		}
```

Comment:
That logic was outdated; I'm commenting it out as a reminder to build the patch reordering. Right now we're simply assuming it comes in order.